pkg/runtime/dispatcher.go

```diff
@@ -121,15 +121,13 @@ type dispatcher struct {
 	resolver     Resolver
 }
 
-// SetResolver installs a new resolver.
+// ChangeResolver installs a new resolver.
 // This function is called when configuration is updated by the user.
 // oldResolver is returned so that it can be reclaimed.
-func (m *dispatcher) SetResolver(rt Resolver) Resolver {
+func (m *dispatcher) ChangeResolver(rt Resolver) {
```

Comment:
SetResolver appears to be clearer about what the function is. ChangeResolver sounds like a single word.